hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/impl/StoreContext.java

```diff
@@ -411,4 +416,8 @@ public RequestFactory getRequestFactory() {
   public boolean isCSEEnabled() {
     return isCSEEnabled;
   }
+
+  public S3ObjectStorageClassFilter s3ObjectsStorageClassFilter() {
```

Comment:
nit: rename to getS3ObjectStorageClassFilter(), and add java docs for the method